app/planqtn_api/planqtn_server.py

```diff
@@ -16,12 +16,21 @@
 )
 
 # Add CORS middleware
+allowed_origins_env = os.getenv("PLANQTN_API_ALLOWED_ORIGINS", "*")
```

Comment:
I'm open to these variables, but then we should wire it through the `htn` tool, so developers can set it during `hack/htn cloud deploy` . Please see https://github.com/planqtn/planqtn/blob/main/DEVELOPMENT.md.